src/Tools/dotnet-monitor/ValidationResultExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.ComponentModel.DataAnnotations;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.Diagnostics.Tools.Monitor
+{
+    internal static class ValidationResultExtensions
+    {
```

Comment:
Is this the correct condition? Doesn't null imply success?

Reply:
Also consider adding some comments here

Reply:
> Is this the correct condition? Doesn't null imply success?

It does, null is explicitly success. I'll add some comments to clarify this